paper-tabs.html

```diff
@@ -133,15 +133,19 @@
         -moz-flex-basis: auto;
         -ms-flex-basis: auto;
         flex-basis: auto;
+        @apply(--layout-horizontal);
       }
 
       #tabsContent.scrollable {
         position: absolute;
+        min-width: 100%;
         white-space: nowrap;
       }
 
-      #tabsContent.horizontal {
-        @apply(--layout-horizontal);
+      #tabsContent.scrollable > ::content > * {
+        -ms-flex-shrink: 0;
+        -webkit-flex-shrink: 0;
+        flex-shrink: 0;
```

Comment:
Any particular reason we need `flex-shrink: 0`? Does the default `flex-basis: auto` not prevent the items from collapsing too much here?